### PR TITLE
Add github action to make sure changelog for @excalidraw/excalidraw is updated

### PR DESCRIPTION
**Pull request by ad1992**
Since `@excalidraw/excalidraw` is released so its imp to keep changelog up to date.
This action will fail if any such file has changed and changelog is not updated.
Later we can automate this process.

The changelog is updated with changes pushed post the last release

### CODE REVIEW

File: scripts/changelog-check.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { exec } = require("child_process");
+
+function changeLogCheck() {
```
Comment (lipis) [inline scripts/changelog-check.js:3]:
```suggestion
const changeLogCheck = () => {
```

File: src/packages/excalidraw/CHANGELOG.MD [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
+Aligns arrowhead schemas [#2517](https://github.com/excalidraw/excalidraw/pull/2517)
+Expand canvas padding based on zoom. [#2515](https://github.com/excalidraw/excalidraw/pull/2515)
+Fix Library Menu Layout [#2502](https://github.com/excalidraw/excalidraw/pull/2502)
+Add Cut to menus [#2511)](https://github.com/excalidraw/excalidraw/pull/2498)
+hide stats and scrollToContent-button when mobile menus open [#2509](https://github.com/excalidraw/excalidraw/pull/2509)
+Hide shortcuts on pickers for mobile [#2508](https://github.com/excalidraw/excalidraw/pull/2508)
+More Arrowheads: dot, bar [#2486](https://github.com/excalidraw/excalidraw/pull/2486)
+Bump ini from 1.3.5 to 1.3.7 in /src/packages/excalidraw [#2500](https://github.com/excalidraw/excalidraw/pull/2500)
+Support CSV graphs and improve the look and feel [#2495](https://github.com/excalidraw/excalidraw/pull/2495)
+
```
Comment (lipis) [inline src/packages/excalidraw/CHANGELOG.MD:17]: Make it as a list.. it's not good in the [preview](https://github.com/excalidraw/excalidraw/blob/aakansha-changelog/src/packages/excalidraw/CHANGELOG.MD)

Comment (ad1992) [reply src/packages/excalidraw/CHANGELOG.MD:17]: My bad. fixed

File: src/packages/excalidraw/CHANGELOG.MD [hunk 3]
```diff
@@ -1,10 +1,22 @@
 # Changelog
 
-## 0.1.0
+### [Unreleased]
 
-First release of `@excalidraw/excalidraw`
+- Aligns arrowhead schemas [#2517](https://github.com/excalidraw/excalidraw/pull/2517)
+- Expand canvas padding based on zoom. [#2515](https://github.com/excalidraw/excalidraw/pull/2515)
+- Fix Library Menu Layout [#2502](https://github.com/excalidraw/excalidraw/pull/2502)
+-Add Cut to menus [#2511)](https://github.com/excalidraw/excalidraw/pull/2498)
```
Comment (lipis) [inline src/packages/excalidraw/CHANGELOG.MD:8]:
```suggestion
- Add Cut to menus [#2511)](https://github.com/excalidraw/excalidraw/pull/2498)
```

File: src/packages/excalidraw/CHANGELOG.MD [hunk 4]
```diff
@@ -1,10 +1,40 @@
...
+- More Arrowheads: dot, bar [#2486](https://github.com/excalidraw/excalidraw/pull/2486)
+- Support CSV graphs and improve the look and feel [#2495](https://github.com/excalidraw/excalidraw/pull/2495)
+
+### Fixes
+- Fix Library Menu Layout [#2502](https://github.com/excalidraw/excalidraw/pull/2502)
+
+### Improvements
+- Expand canvas padding based on zoom. [#2515](https://github.com/excalidraw/excalidraw/pull/2515)
+- Hide shortcuts on pickers for mobile [#2508](https://github.com/excalidraw/excalidraw/pull/2508)
+- hide stats and scrollToContent-button when mobile menus open [#2509](https://github.com/excalidraw/excalidraw/pull/2509)
```
Comment (lipis) [inline src/packages/excalidraw/CHANGELOG.MD:28]:
```suggestion
- Hide stats and scrollToContent-button when mobile menus open [#2509](https://github.com/excalidraw/excalidraw/pull/2509)
```

**Comment by dwelle** [comment]
As I understand it, this will run on master once a PR is merged, and thus fail after the PR landed? If so, shouldn't we run this in the PR itself so that we can update the changelog within the same PR?

**Comment by ad1992** [comment]
@dwelle this will run on PR always as you can see the checks and fail if changelog not updated. On the master branch, it's just added as a safe check and will pass if the changelog has been added already.

**Comment by dwelle** [comment]
I see we're doing the same for `lint.yml` etc.

That said, wasn't the workflow supposed to run for the last commit that landed on master 15 minutes ago? https://github.com/excalidraw/excalidraw/commit/81c17a56fb6d8ad26480d06586cb5b2e0ea406d5

**Comment by ad1992** [comment]
> I see we're doing the same for `lint.yml` etc.
> 
> That said, wasn't the workflow supposed to run for the last commit that landed on master 15 minutes ago? [81c17a5](https://github.com/excalidraw/excalidraw/commit/81c17a56fb6d8ad26480d06586cb5b2e0ea406d5)
> 
> EDIT: maybe I still have a wrong mental model. I'll need to read up on the workflow events.

This action will always check the diff between the current PR with master branch and decide whether the changelog needs to be updated for this diff.

**Review by dwelle — APPROVED** [review]

**Pull request merged**